### ISSUE TITLE
A possible datasource other then ogcio

### ISSUE THREAD

**Issue by ylchan87** [g0vhk-io/HKAddressParser#57]
https://geodata.gov.hk/gs/locationSearchAPI

Seems to support building name

**Comment by cswbrian** [comment]
Yes, pleace check [this link](https://www.info.gov.hk/gia/general/201812/10/P2018121000352.htm?fbclid=IwAR0oM5qMN2ExkU1CW-sYoAWd6O2dgHrniOxoKuHHzfJwtw4km7IGwp0DPDs) for more information, I think we can try to use some of the APIs.
